components/Header/index.vue

```diff
@@ -1,65 +1,74 @@
 <template>
-
-
-    <el-col  :span="3"  style="float:left; " >
-    <el-menu 
+  <el-col :span="3" style="float:left; ">
+    <el-menu
       default-active="1"
       class="el-menu-vertical-demo"
       background-color="#545c64"
       text-color="#fff"
-      active-text-color="#ffd04b">
-     
+      active-text-color="#ffd04b"
+    >
       <el-menu-item index="1">
         <i class="el-icon-menu"></i>
-            <span class="title"></span>
-            <nuxt-link  to="/" class="link">Dashboard</nuxt-link>
+        <span class="title"></span>
+        <nuxt-link to="/" class="link">Dashboard</nuxt-link>
       </el-menu-item>
       <el-menu-item index="2">
         <i class="el-icon-edit"></i>
-             <span class="title"></span>
-        <nuxt-link to="/HC/operations" class="link" >
-        Operations</nuxt-link> 
+        <span class="title"></span>
+        <nuxt-link to="/HC/operations">Operations</nuxt-link>
       </el-menu-item>
 
       <el-menu-item index="3">
-             <span class="title"></span>
+        <span class="title"></span>
         <i class="el-icon-view b"></i>
-        <nuxt-link  to="/HC" class="link">Employees</nuxt-link>
-       
-       
+        <nuxt-link to="/HC" class="link">Employees</nuxt-link>
       </el-menu-item>
     </el-menu>
   </el-col>
-
-
-
-
-
-
 </template>
 
 <script>
-import backgroundUrl from '~/assets/img/firstbg.jpg'
-  export default {
-    data() {
-    return { backgroundUrl }
+import backgroundUrl from "~/assets/img/firstbg.jpg";
+export default {
+  data() {
+    return { backgroundUrl };
   }
-    }
-  
+};
 </script>
 
 <style>
```

Comment:
You should add `scoped` here as you're not encapsulating your styles. See:

- https://vue-loader.vuejs.org/guide/scoped-css.html
- https://vuejs.org/v2/style-guide/

Reply:
Added scoped in style tags